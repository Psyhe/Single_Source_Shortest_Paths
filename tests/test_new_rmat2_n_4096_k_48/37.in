4096 3161 3245
3161 3165 226
3161 3157 199
3161 1073 113
3161 2577 81
3161 3289 157
3161 8 204
3161 3417 29
3161 2521 31
3161 3099 34
3161 3655 245
3161 3153 173
3161 1153 55
3161 3445 131
3162 3210 65
3162 3294 135
3162 3195 137
3162 2262 188
3162 94 116
3162 3338 171
3162 1578 134
3162 3082 216
3162 3675 33
3162 2132 28
3163 2554 144
3163 1723 161
3163 3099 192
3163 3578 216
3163 3627 42
3163 1307 53
3163 59 207
3163 2203 72
3163 3278 31
3163 3707 212
3163 3177 31
3164 3148 216
3164 1234 37
3164 3160 52
3164 3188 173
3164 1108 204
3164 3613 10
3164 2120 32
3164 1116 70
3164 1562 251
3164 1116 34
3164 3948 118
3164 2128 4
3164 3156 190
3165 3145 199
3165 95 29
3165 3158 104
3165 1093 13
3165 3700 16
3165 3161 226
3165 2143 143
3165 3677 195
3165 3167 66
3165 3357 251
3165 1101 64
3165 2652 163
3165 93 136
3165 2133 5
3165 3704 77
3165 93 173
3166 2124 87
3166 1118 140
3166 3657 148
3166 1118 234
3166 3418 174
3167 3165 66
3167 3195 226
3167 3151 142
3167 95 100
3168 3424 133
3168 1120 220
3168 1056 24
3168 97 179
3168 1120 147
3168 1080 158
3168 3202 138
3168 2408 238
3168 2144 50
3168 3554 117
3168 1056 9
3168 3680 228
3168 544 9
3168 1120 230
3168 2148 58
3168 3682 177
3168 3684 154
3168 3172 246
3168 3320 154
3168 3428 35
3168 3072 243
3168 2280 178
3168 2144 4
3168 356 37
3168 3300 70
3168 3680 2
3168 3192 39
3168 3616 79
3169 3115 47
3169 3140 244
3169 1088 65
3169 3425 121
3169 3297 164
3169 2760 150
3169 1121 172
3169 3776 67
3169 97 88
3169 3297 7
3169 3681 241
3169 1385 139
3169 96 14
3169 320 76
3169 3433 10
3169 3233 97
3170 3178 122
3170 3174 87
3170 1187 28
3170 2146 235
3170 3362 55
3170 2532 39
3170 3378 162
3170 3300 68
3170 3182 226
3170 3299 53
3170 3306 195
3170 3298 129
3170 1122 119
3170 1120 94
3171 2147 243
3171 3174 224
3171 3121 112
3171 1126 84
3171 3283 88
3171 3299 146
3171 1099 182
3171 1060 126
3171 1103 124
3171 3111 97
3171 3298 145
3171 3699 166
3172 36 101
3172 100 143
3172 3168 246
3172 3942 102
3172 3404 247
3172 3702 229
3172 1361 148
3172 3430 87
3172 1092 108
3172 2148 183
3172 3156 111
3172 3156 213
3172 2156 126
3172 68 145
3172 484 82
3172 3108 106
3172 3444 202
3172 2144 178
3172 324 118
3172 2660 55
3172 1390 255
3173 3072 5
3173 3111 252
3173 2084 106
3173 2149 50
3173 2153 252
3173 2117 43
3173 3189 235
3174 3630 208
3174 3170 87
3174 3126 244
3174 2408 59
3174 3171 224
3174 1024 243
3174 1604 15
3174 100 231
3174 1142 221
3174 1208 118
3174 3108 233
3175 3381 92
3175 2087 95
3175 2150 219
3175 3148 14
3175 3378 184
3175 3109 11
3175 1090 219
3175 3111 21
3175 2420 82
3175 2631 205
3175 3503 0
3176 2152 53
3176 3660 14
3176 3833 52
3176 2792 121
3176 3432 54
3176 3290 12
3176 3625 242
3176 1632 155
3176 3180 45
3176 1312 203
3176 2634 35
3176 72 7
3176 104 224
3176 3116 147
3176 3177 227
3176 3944 154
3176 3552 120
3176 2281 168
3176 3082 52
3177 1225 106
3177 3256 2
3177 3163 31
3177 3695 98
3177 1169 180
3177 1395 30
3177 2149 100
3177 3625 174
3177 3145 186
3177 1128 147
3177 3176 227
3177 3432 40
3177 3917 169
3177 3405 43
3177 3708 150
3178 1318 175
3178 2408 159
3178 3298 36
3178 2120 13
3178 2138 145
3178 3194 218
3178 1146 120
3178 3438 143
3178 1130 144
3178 3306 230
3178 1059 46
3178 1098 160
3178 3182 185
3178 3170 122
3178 3119 7
3178 3146 61
3179 3242 132
3179 1234 212
3179 3138 127
3179 1133 117
3179 1274 146
3179 3191 144
3179 2122 161
3179 3323 92
3180 1391 229
3180 1196 233
3180 3673 111
3180 1100 186
3180 2092 146
3180 3176 45
3180 3277 124
3180 2668 75
3180 3198 57
3180 2412 250
3180 3692 138
3180 3112 240
3180 1580 225
3180 3106 119
3180 2156 7
3180 1116 177
3180 2168 111
3180 3628 204
3180 110 121
3181 2125 118
3181 1384 164
3181 3117 172
3181 3133 194
3181 3137 114
3181 101 116
3181 2120 187
3181 3141 113
3181 2399 196
3181 75 164
3181 1120 58
3182 3241 41
3182 1064 135
3182 3178 185
3182 1514 43
3182 3190 34
3182 3170 226
3183 3143 205
3183 3087 25
3183 3241 94
3183 2116 42
3183 3021 139
3183 3107 124
3183 3590 122
3183 1276 233
3184 2160 6
3184 2416 189
3184 3824 63
3184 1137 108
3184 3440 200
3184 1136 157
3184 2160 178
3184 2160 89
3184 3244 175
3184 3377 111
3184 3284 30
3184 3612 177
3184 2272 213
3184 3196 5
3185 1145 99
3185 1648 63
3185 3121 239
3185 2163 118
3185 3141 180
3185 353 222
3185 1136 77
3185 3105 33
3185 3129 131
3185 1136 143
3185 3320 176
3185 3543 252
3186 2074 51
3186 3154 228
3186 2930 204
3186 3301 5
3186 2098 130
3186 3446 73
3186 368 209
3186 3586 167
3186 2166 240
3186 3922 60
3186 3106 139
3186 114 212
3187 1250 41
3187 1139 216
3187 1147 61
3187 3137 180
3187 3194 23
3188 3636 92
3188 1076 106
3188 2100 112
3188 1298 207
3188 3098 30
3188 1394 35
3188 3412 253
3188 3313 221
3188 3254 72
3188 3670 47
3188 2134 78
3188 2276 92
3188 2412 57
3188 3348 58
3188 3089 169
3188 3164 173
3188 726 8
3188 96 141
3188 1140 50
3189 3708 50
3189 86 145
3189 2097 19
3189 3825 96
3189 3173 235
3189 3125 15
3189 3441 132
3189 2237 66
3189 1109 160
3190 1106 38
3190 3637 91
3190 1618 2
3190 3182 34
3190 3130 11
3190 3194 224
3190 2660 34
3190 3668 69
3191 98 102
3191 3703 238
3191 3639 238
3191 3179 144
3192 2109 74
3192 3168 39
3192 2105 221
3192 3416 145
3192 3288 253
3192 1122 203
3193 440 58
3193 3089 110
3193 3385 177
3194 3312 250
3194 3451 153
3194 3190 224
3194 890 78
3194 3178 218
3194 3435 32
3194 3518 133
3194 106 3
3194 3187 23
3194 2136 56
3195 3115 213
3195 3155 88
3195 1529 4
3195 3162 137
3195 3167 226
3195 2170 207
3195 3347 224
3196 3184 5
3196 2133 230
3196 3198 239
3196 172 129
3196 2108 15
3196 3100 142
3197 1373 222
3197 3837 198
3198 3448 192
3198 3583 174
3198 2170 139
3198 2106 244
3198 3199 254
3198 3418 181
3198 3196 239
3198 3180 57
3198 3690 49
3198 2294 155
3199 3667 15
3199 3198 254
3199 2170 0
3200 3217 133
3200 1152 108
3200 3212 90
3200 3736 70
3200 1097 249
3200 1088 83
3200 512 11
3200 1152 80
3200 3076 57
3200 1160 96
3200 2209 155
3200 2688 207
3200 2240 147
3200 1185 66
3200 1032 253
3200 1152 146
3200 3230 175
3200 1176 45
3200 2208 126
3200 1473 58
3200 3360 33
3200 2628 43
3200 3618 51
3200 3234 12
3200 2048 154
3200 1289 60
3200 128 138
3200 2496 235
3200 3212 0
3200 3269 144
3200 3077 64
3201 3237 100
3201 3585 230
3201 1491 50
3201 2193 251
3201 3265 201
3201 3777 193
3201 3073 125
3201 3081 230
3201 2569 96
3201 3215 33
3201 3585 235
3201 2476 117
3201 3073 85
3201 1668 98
3201 3081 7
3201 3209 70
3201 3217 151
3201 194 155
3201 2179 132
3201 1040 156
3201 1040 132
3201 1157 109
3201 3777 248
3201 3525 3
3201 3218 154
3201 1153 111
3201 3265 9
3201 3746 160
3202 2321 46
3202 3216 203
3202 2563 124
3202 2112 128
3202 3282 250
3202 1216 71
3202 194 255
3202 1434 102
3202 3168 138
3202 3226 200
3202 3586 125
3202 1202 17
3202 3266 124
3202 3080 148
3202 2947 34
3202 3713 136
3202 2195 169
3202 2246 175
3202 1154 183
3202 3242 48
3202 3158 117
3202 3530 178
3202 2177 143
3202 3238 148
3202 3104 95
3202 2049 252
3202 1082 225
3203 3971 230
3203 3730 229
3203 1424 189
3203 3731 54
3203 3713 127
3203 3075 210
3203 2049 95
3203 2176 244
3203 3359 96
3203 1026 205
3203 2563 150
3203 3299 98
3203 3075 53
3203 3594 87
3203 2061 38
3203 1681 189
3203 2561 69
3203 1030 132
3203 1667 44
3203 3213 132
3204 3228 143
3204 3236 122
3204 3652 248
3204 3072 183
3204 3637 21
3204 3076 202
3204 4 45
3204 3108 173
3204 3108 23
3204 3216 174
3204 1158 183
3204 3236 184
3204 3341 37
3204 3460 144
3204 3206 239
3204 1158 135
3204 1157 160
3204 1156 203
3204 3814 61
3204 3084 85
3204 3269 190
3204 3106 82
3204 2214 174
3204 1156 143
3204 3205 251
3204 2180 84
3204 2176 220
3204 2176 98
3204 3379 27
3205 3269 156
3205 3220 201
3205 3457 126
3205 1158 163
3205 1158 220
3205 1093 0
3205 3229 235
3205 3269 210
3205 3304 111
3205 132 203
3205 3085 67
3205 3204 251
3205 3461 149
3205 3149 60
3205 3717 237
3205 3332 39
3205 3334 113
3206 1182 175
3206 1216 40
3206 3204 239
3206 14 207
3206 3270 144
3206 2182 3
3206 3399 85
3206 516 8
3206 2198 118
3206 3222 155
3206 2182 85
3206 3238 168
3207 140 156
3207 2222 143
3207 3093 64
3207 2439 124
3207 3398 58
3207 2090 233
3207 3526 240
3207 3223 134
3207 1155 148
3207 710 106
3208 1672 24
3208 2184 183
3208 3800 118
3208 3240 144
3208 200 90
3208 1160 175
3208 170 58
3208 3596 203
3208 2216 165
3208 1185 149
3208 2186 31
3208 2696 243
3208 2184 229
3208 3458 219
3208 2184 22
3208 3242 29
3208 3212 66
3208 3465 32
3208 3224 5
3208 3832 26
3208 1194 190
3208 3744 86
3208 1673 134
3208 3656 97
3208 2058 10
3208 3224 176
3208 160 149
3208 1282 206
3208 3464 21
3208 3080 149
3208 2200 40
3208 72 14
3208 3088 145
3208 1056 176
3209 3714 2
3209 3082 57
3209 2697 226
3209 3368 37
3209 2185 206
3209 2057 25
3209 3213 148
3209 3113 13
3209 3973 157
3209 1416 193
3209 2233 199
3209 2179 195
3209 2793 74
3209 3241 136
3209 3201 70
3209 3240 240
3209 3337 194
3209 683 152
3209 1033 229
3209 2064 255
3209 2177 240
3210 2219 194
3210 1674 79
3210 3306 45
3210 174 20
3210 3242 65
3210 411 194
3210 2254 153
3210 603 31
3210 3231 218
3210 3214 96
3210 3162 65
3210 2219 111
3210 155 224
3210 2054 178
3210 1956 87
3210 1216 200
3210 130 235
3210 3298 183
3211 1155 83
3211 2181 162
3211 138 4
3211 3075 25
3211 1434 192
3211 2443 19
3211 3215 197
3211 3593 238
3211 2691 143
3211 3587 4
3211 1155 237
3211 2459 165
3212 396 9
3212 3208 66
3212 1412 226
3212 1156 31
3212 3484 144
3212 2253 99
3212 3270 36
3212 2238 173
3212 1820 209
3212 1164 247
3212 1165 139
3212 159 9
3212 2252 137
3212 3072 14
3212 2180 33
3212 3200 0
3212 1164 199
3212 3076 78
3212 1036 128
3212 3141 67
3212 1230 72
3212 3200 90
3212 140 155
3212 2702 191
3213 3228 177
3213 2639 8
3213 3225 250
3213 2571 70
3213 3241 80
3213 3209 148
3213 1672 175
3213 3333 60
3213 2060 160
3213 3325 101
3213 2497 86
3213 2761 66
3213 3203 132
3214 3242 6
3214 132 127
3214 2188 52
3214 3468 52
3214 3210 96
3214 3294 133
3214 1230 171
3214 1186 186
3214 3242 33
3214 3074 250
3214 3590 151
3214 2186 162
3214 1418 215
3214 2314 31
3215 1035 243
3215 2182 121
3215 3151 148
3215 3479 39
3215 3534 157
3215 3201 33
3215 3726 108
3215 3211 197
3216 3344 88
3216 1168 202
3216 2073 6
3216 3217 62
3216 216 174
3216 3075 97
3216 3204 174
3216 3090 82
3216 3281 108
3216 2097 68
3216 1044 110
3216 2728 108
3216 3296 118
3216 1168 57
3216 3221 62
3216 1178 229
3216 3428 101
3216 3720 150
3216 3220 163
3216 3202 203
3216 2386 234
3216 388 162
3216 3588 12
3216 16 215
3216 132 244
3216 2196 241
3216 3301 192
3216 2352 100
3217 3729 253
3217 3200 133
3217 3201 151
3217 3072 66
3217 2432 60
3217 129 185
3217 3224 44
3217 2065 59
3217 3221 178
3217 3745 103
3217 3249 140
3217 1104 2
3217 3219 226
3217 3216 62
3217 2788 20
3217 2833 97
3217 1465 18
3217 1185 152
3217 2105 102
3217 1105 155
3217 145 136
3217 3477 123
3217 1137 237
3217 3733 205
3218 1174 6
3218 130 218
3218 1170 134
3218 2443 202
3218 3458 202
3218 1222 236
3218 3201 154
3218 2256 207
3218 3234 115
3218 2710 18
3218 3824 128
3218 3224 7
3219 3361 124
3219 1171 155
3219 2243 143
3219 1047 81
3219 3607 2
3219 3539 7
3219 3105 121
3219 2067 150
3219 657 37
3219 151 59
3219 3739 89
3219 3238 114
3219 3475 137
3219 3217 226
3219 3090 144
3219 3299 38
3219 2859 63
3219 2234 105
3220 2832 185
3220 3205 201
3220 3216 163
3220 2192 167
3220 3222 147
3220 3732 77
3220 1684 140
3220 3090 115
3220 2196 157
3220 3486 119
3220 1170 237
3220 3286 186
3220 3732 1
3220 2436 122
3220 3476 55
3221 3217 178
3221 3517 129
3221 3092 159
3221 2773 32
3221 3216 62
3221 1044 186
3221 3827 66
3221 1030 57
3221 1545 186
3221 3719 167
3221 2261 95
3222 4054 73
3222 3734 84
3222 2071 90
3222 1173 237
3222 2052 178
3222 3094 112
3222 468 148
3222 3206 155
3222 3158 45
3222 2294 177
3222 3220 147
3222 3476 253
3223 3093 150
3223 2198 18
3223 2582 12
3223 2206 237
3223 2181 51
3223 3735 9
3223 3207 134
3223 2183 122
3223 3415 61
3223 3230 1
3223 1171 13
3223 2207 26
3224 3481 57
3224 3280 203
3224 1243 86
3224 3096 155
3224 1168 160
3224 3217 44
3224 3096 184
3224 3208 5
3224 3208 176
3224 1274 165
3224 3218 7
3224 3482 27
3224 3228 199
3224 3568 45
3224 1433 135
3224 3284 55
3225 3609 54
3225 315 175
3225 3213 250
3225 2261 3
3225 1161 161
3225 3480 106
3225 3473 222
3226 2209 145
3226 3790 194
3226 2266 18
3226 3202 200
3226 166 42
3226 3550 111
3226 1179 192
3226 1178 91
3226 3482 63
3226 690 10
3227 3275 46
3227 3338 222
3227 1547 184
3227 2049 111
3227 2715 149
3227 2204 76
3227 3261 171
3228 3084 209
3228 3100 51
3228 3204 143
3228 2716 29
3228 3213 177
3228 1438 5
3228 3224 199
3228 3318 62
3228 3805 180
3228 3828 12
3228 2525 118
3228 3740 150
3228 2716 211
3228 276 49
3229 2204 88
3229 2749 179
3229 221 138
3229 3137 54
3229 3103 218
3229 3097 196
3229 3097 184
3229 2077 4
3229 3092 44
3229 2569 62
3229 3477 112
3229 3093 151
3229 3205 235
3229 1053 146
3230 3742 179
3230 1748 206
3230 3200 175
3230 3223 1
3230 2334 214
3230 3100 120
3230 3231 191
3231 3230 191
3231 3263 38
3231 1157 23
3231 2463 41
3231 1238 240
3231 3773 90
3231 1037 38
3231 1355 23
3231 3210 218
3232 1664 151
3232 3234 124
3232 3104 244
3232 2072 31
3232 3252 45
3232 3084 87
3232 2090 12
3232 3073 48
3232 2208 197
3232 3234 113
3232 1185 99
3232 1956 33
3232 3488 252
3232 3234 203
3232 1184 185
3232 3233 187
3232 1184 84
3232 3520 49
3232 3104 15
3232 3233 212
3232 3234 131
3232 2082 161
3232 1440 247
3232 3104 235
3232 2176 158
3232 106 198
3232 3238 40
3233 160 132
3233 3249 139
3233 3752 199
3233 3169 97
3233 3232 212
3233 2209 155
3233 3073 215
3233 2465 241
3233 3105 150
3233 3237 4
3233 1192 38
3233 2433 27
3233 3122 237
3233 2465 214
3233 3104 192
3233 3731 152
3233 2052 31
3233 161 237
3233 3745 199
3233 161 34
3233 3232 187
3233 3329 116
3233 3361 2
3234 3302 223
3234 2186 28
3234 3218 115
3234 2275 88
3234 3232 131
3234 2722 10
3234 3240 167
3234 2210 167
3234 2178 58
3234 3200 12
3234 51 243
3234 1216 74
3234 3232 124
3234 3242 11
3234 3236 131
3234 3474 7
3234 3422 45
3234 2464 27
3234 1186 63
3234 1168 164
3234 3232 113
3234 1206 14
3234 3970 50
3234 2491 94
3234 2344 62
3234 3232 203
3234 2178 89
3235 3107 215
3235 2051 250
3235 1891 131
3235 3090 81
3235 1441 204
3235 3109 209
3235 1154 166
3235 3243 145
3235 3239 145
3235 2473 108
3235 2980 118
3235 3299 144
3235 2315 202
3235 154 226
3236 3748 215
3236 1156 183
3236 2084 240
3236 2208 170
3236 3744 213
3236 3302 192
3236 693 3
3236 1300 183
3236 1062 189
3236 3108 229
3236 3204 122
3236 3476 90
3236 3300 15
3236 1212 81
3236 2229 118
3236 3234 131
3236 3488 158
3236 3204 184
3236 1668 178
3236 3412 229
3236 3108 174
3237 1061 109
3237 2597 122
3237 2213 189
3237 3504 90
3237 3493 171
3237 1153 110
3237 3201 100
3237 3717 20
3237 2181 36
3237 1204 242
3237 116 111
3237 3880 135
3237 3300 237
3237 134 211
3237 2276 119
3237 3233 4
3237 3271 150
3237 3461 22
3237 3628 5
3237 3747 209
3237 2148 248
3238 3749 136
3238 2094 202
3238 3219 114
3238 3261 133
3238 1510 110
3238 3202 148
3238 678 191
3238 3239 136
3238 2208 195
3238 3247 209
3238 3206 168
3238 3110 3
3238 3232 40
3239 1157 150
3239 3238 136
3239 2277 121
3239 3247 158
3239 1026 198
3239 3235 145
3239 1191 199
3239 3241 113
3239 1187 247
3240 3992 33
3240 3120 70
3240 2272 60
3240 3372 228
3240 3496 49
3240 3209 240
3240 2184 186
3240 1224 4
3240 3208 144
3240 1189 246
3240 3304 31
3240 3272 3
3240 2733 240
3240 44 114
3240 3244 110
3240 104 129
3240 33 161
3240 3258 228
3240 3234 167
3240 1676 179
3241 3239 113
3241 1065 94
3241 3114 80
3241 2189 49
3241 1153 51
3241 1161 22
3241 3213 80
3241 2153 141
3241 3497 169
3241 163 65
3241 1313 113
3241 3209 136
3241 3182 41
3241 3183 94
3241 2224 217
3242 3214 33
3242 3466 179
3242 3210 65
3242 106 212
3242 3202 48
3242 1056 101
3242 3259 66
3242 3214 6
3242 1723 119
3242 2440 230
3242 3244 23
3242 3208 29
3242 3304 38
3242 3112 153
3242 3234 11
3242 1450 21
3242 1450 66
3242 3179 132
3242 3298 17
3242 1186 34
3243 168 207
3243 3755 50
3243 3755 83
3243 2477 156
3243 3371 12
3243 2746 58
3243 3660 30
3243 1193 40
3243 3082 0
3243 3235 145
3243 3113 42
3243 2202 35
3244 1068 249
3244 3096 123
3244 45 87
3244 1196 130
3244 3240 110
3244 3812 163
3244 756 225
3244 1068 19
3244 3124 54
3244 3076 74
3244 3242 23
3244 2218 7
3244 3184 175
3244 3272 170
3245 2253 248
3245 3113 175
3245 3631 155
3245 13 184
3245 3497 192
3245 2793 37
3245 2152 194
3245 3247 77
3245 2732 93
3245 1419 13
3245 1189 51
