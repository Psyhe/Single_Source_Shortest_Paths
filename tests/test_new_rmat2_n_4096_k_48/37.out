75
45
83
21
52
130
118
17
18
85
131
104
27
61
29
37
80
42
144
52
71
72
82
26
65
46
78
63
77
38
113
56
114
55
112
31
318
94
57
26
49
83
92
58
39
30
104
44
51
66
23
26
114
49
82
39
31
56
71
67
77
126
40
49
95
63
122
76
27
41
55
39
47
32
109
29
51
79
182
51
70
43
45
54
46
